packages/network-controller/src/rpc-service/rpc-service.ts

```diff
@@ -336,11 +338,11 @@ export class RpcService implements AbstractRpcService {
    * @param fetchOptions - An options bag for {@link fetch} which further
    * specifies the request.
    * @returns The decoded JSON-RPC response from the endpoint.
-   * @throws A "method not found" error if the response status is 405.
-   * @throws A rate limiting error if the response HTTP status is 429.
-   * @throws A timeout error if the response HTTP status is 503 or 504.
-   * @throws A generic error if the response HTTP status is not 2xx but also not
-   * 405, 429, 503, or 504.
+   * @throws A 401 error if the response status is 401.
```

Comment:
?
```suggestion
   * @throws A "Unauthorized" error if the response status is 401.
```

Reply:
Good call. Updated in [957d6b4](https://github.com/MetaMask/core/pull/5923/commits/957d6b446425f7c4bc1fa2550ba6625b4f97e33c).